dubbo-configcenter/dubbo-configcenter-nacos/src/main/java/org/apache/dubbo/configcenter/support/nacos/NacosDynamicConfiguration.java

```diff
@@ -93,24 +97,14 @@ private ConfigService buildConfigService(URL url) {
         return configService;
     }
 
-    public void publishNacosConfig(String key, String value) {
+    public void publishNacosConfig(String key, String group, String value) {
```

Comment:
I don't find the usage of this method.

Reply:
It is in test code, NacosDynamicConfigurationTest

Reply:
It is in test code, see `NacosDynamicConfigurationTest`